packages/theming/theme-tokens/src/getAliasTokens.macos.ts

```diff
@@ -1,14 +1,25 @@
 import * as macOSLightAliasTokens from './light-macos/tokens-alias';
 import * as macOSDarkAliasTokens from './dark-macos/tokens-alias';
+import * as macOSLightHCAliasTokens from './light-high-contrast-macos/tokens-alias';
+import * as macOSDarkHCAliasTokens from './dark-high-contrast-macos/tokens-alias';
 import * as hcAliasTokens from './highContrast/tokens-alias';
 import { AppearanceOptions } from '@fluentui-react-native/theme-types';
 import { assertNever } from 'assert-never';
+import { isHighContrastMode } from '../../apple-theme/src/createAppleTheme.macos';
```

Comment:
I would prefer this get passed in as a variable than have the theme-tokens package import something from the apple-theme package. That way the theme-tokens package doesn't explicitly depend on any platform theme.

Reply:
Yeah and I noticed that this is breaking the build too :/